### PR TITLE
langchain-snowflake 0.1.3 :snowflake:

### PR DESCRIPTION
**Pull request by tombenes**
langchain-snowflake 0.1.3 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5806]
- dev_url:        https://github.com/langchain-ai/langchain-snowflake
- conda_forge:    None
- pypi:           https://pypi.org/project/langchain-snowflake/0.1.3
- pypi inspector: n/a

### Explanation of changes:

- new version number

[PKG-5806]: https://anaconda.atlassian.net/browse/PKG-5806?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - poetry-core >=1.0.0
+    - pip
+  run:
+    - python
+    - langchain-core >=0.3.0,<0.4.0
+    - snowflake >=0.8.0,<1.0
+
+test:
+  imports:
+    - langchain_snowflake
```
Comment (lorepirri) [inline recipe/meta.yaml:29]:
```suggestion
    - langchain_snowflake
    - langchain_snowflake.CortexSearchRetriever
    - langchain_snowflake.CortexSearchRetrieverError
```

Comment (lorepirri) [reply recipe/meta.yaml:29]: https://github.com/langchain-ai/langchain-snowflake/blob/main/libs/snowflake/tests/unit_tests/test_imports.py

**Review by lorepirri — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - langchain_snowflake.CortexSearchRetriever
+    - langchain_snowflake.CortexSearchRetrieverError
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
    - langchain_snowflake.retrievers.CortexSearchRetriever
    - langchain_snowflake.retrievers.CortexSearchRetrieverError
```

**Review by lorepirri — APPROVED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (lorepirri) [reply recipe/meta.yaml]: sorry about the misleading suggestion: they are not re-exported in the init

**Review by lorepirri — APPROVED** [review]
LGTM, thanks for adding the imports as a command!

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**